src/bundle/Resources/views/themes/admin/link_manager/list.html.twig

```diff
@@ -28,7 +28,7 @@
                 }}) }}
 
                 <span class="input-group-btn">
-                    <button class="btn btn-primary" title="{{ 'url.search'|trans|desc("Search") }}">
```

Comment:
Shouldn't this be removed from extracted translations?

Reply:
This translation is used below